src/pages/VerdictPage.tsx

```diff
@@ -343,15 +361,27 @@ export default function VerdictPage() {
             </SubmissionTableElem>
           </Grid>
           <Box sx={{ p: 2, display: 'flex', flexWrap: 'wrap', gap: 2 }}>
-            {testCases?.map((status) => {
-              if (status == 3) {
-                return <CorrectIcon />
-              }
-              if (status > 3) {
-                return <WrongAnswerIcon />
-              }
-              return <PendingIcon />
-            })}
+            {testCases?.map((status) => (
+              <Tooltip
+                title={verdictInfo[status].description}
```

Comment:
I really like this. But how about different test cases with different verdicts. This is in regards to how Judge0 is sending the verdict. Cause if there is a wrong answer test case and time limit exceeded test case, and the overall verdict being only one of that, then that would be mean the tooltip for the other test case is incorrect.

Reply:
I don't think that's true. This tooltip should show each unique test case's 'verdict', unless the code is wrong somewhere.

Reply:
alright I missed that, cool 👍